templates/patterns/card/pattern-card.html.twig

```diff
@@ -37,9 +37,8 @@
     right_col_classes: 'col-8',
     gutter: 0,
   } : {},
-  'title': (title is not empty) ? {
-    'content': title,
-  } : {},
+  'title': title,
+  'title_tag': 'h5',
```

Comment:
I talked with front-end.
The idea is that card titles should now be h1. Semantically this is ok because they are inside <article>, so they only apply to the article.
This means we don't need to set it to 'h5'.